drupal/acquia/update_hp_paragraphs.sh

```diff
@@ -0,0 +1,5 @@
+
+#Run composer update on all local codebases. 
+composer update
```

Comment:
If you run composer update here, you do not need to run it again at howard/howard_paragraphs

Reply:
ie, running composer update, without a specific package, updates "everything", including howard_paragraphs